docs/topics/installing-cli-tool.adoc

```diff
@@ -24,24 +23,52 @@ Podman provides a command-line interface (CLI) familiar to anyone who has used t
 For more information on installing and using Podman, see link:https://podman.io/docs/installation[Podman installation instructions].
 ====
 
+.Installation options
+
+You can run either of the following two options to install {ProductShortName} {CLINameTitle}:
+
+* Downloading the appropriate {ProductShortName} {CLINameTitle} `.zip` file, according to your operating system, and installing by using the command line.
+* Running Podman on your computer and installing {ProductShortName} {CLIName} using Podman commands.
+
++
+[WARNING]
+====
+Although installation using Podman is possible, downloading and installing the {ProductShortName} {CLIName} `.zip` file is the preferred installation.
+====
+
+
 [id="installing-downloadable-cli-zip_{context}"]
 == Installing the {CLINameTitle} `.zip` file
 
 .Procedure
 
 To install using the downloadable `.zip` file:
 
-. Navigate to the link:{DevDownloadPageURL}[{ProductShortName} Download page] and download the OS-specific CLI file or the `src` file:
+. Navigate to the link:https://developers.redhat.com/products/mta/download[{ProductShortName} Download page] and download the OS-specific {ProductShortName} CLI file or the `mta-{ProductVersion}-cli-src` file:
+
++
+* Migration Toolkit {CLINameTitle} - {CLINameTitle} for Linux x86_64
+* Migration Toolkit {CLINameTitle} - {CLINameTitle} for Linux aarch64
+* Migration Toolkit {CLINameTitle} - {CLINameTitle} for macOS x86_64
+* Migration Toolkit {CLINameTitle} - {CLINameTitle} for macOS aarch64
+* Migration Toolkit {CLINameTitle} - {CLINameTitle} for Windows x86_64
+* Migration Toolkit {CLINameTitle} - {CLINameTitle} for Windows aarch64
+* Migration Toolkit {CLINameTitle} - {CLINameTitle} for {ProductShortName} source Code
+
 +
-* {ProductShortNameLower}-{ProductVersion}-cli-linux.zip
-* {ProductShortNameLower}-{ProductVersion}-cli-macos.zip
-* {ProductShortNameLower}-{ProductVersion}-cli-windows.zip
-* {ProductShortNameLower}-{ProductVersion}-cli-src.zip
+[NOTE]
+====
+Do not download and install the {ProductShortName} source code file if your computer runs one of the above operating systems.
```

Comment:
💡 RedHat.TermsSuggestions 1(101-106) Depending on the context, consider using 'earlier', 'previous', 'preceding', or 'before' rather than 'above'.